mirgecom/eos.py

```diff
@@ -139,11 +139,11 @@ def __init__(self, gamma=1.4, gas_const=287.1):
         self._gamma = gamma
         self._gas_const = gas_const
 
-    def gamma(self, *, cv: ConservedVars = None):
+    def gamma(self):
         """Get specific heat ratio Cp/Cv."""
         return self._gamma
 
-    def gas_const(self, *, cv: ConservedVars = None):
```

Comment:
How come you removed the `cv` arguments? Now the `IdealGasEOS` doesn't comply with the documented interface. Is there a reason to error if it is passed a `cv`?

Reply:
🤷 Beats me.  Changed it back. (bcafb6e)